_schemaV2.graphql

```diff
@@ -15458,6 +15468,49 @@ type updateNotificationPreferencesMutationPayload {
   ): [NotificationPreference!]!
 }
 
+type UpdatePartnerArtworksMutationFailure {
+  mutationError: GravityMutationError
+}
+
+input UpdatePartnerArtworksMutationInput {
+  # Whether Artsy domestic shipping should be enabled
+  artsyShippingDomestic: Boolean
+
+  # Whether Artsy international shipping should be enabled
+  artsyShippingInternational: Boolean
+  clientMutationId: String
+
+  # ID of the partner
+  id: String!
+
+  # The partner location ID to assign
+  location: String
+}
+
+type UpdatePartnerArtworksMutationPayload {
+  clientMutationId: String
+  partnerArtworksOrError: UpdatePartnerArtworksMutationType
+}
+
+type UpdatePartnerArtworksMutationSuccess {
+  partnerArtworks: UpdatePartnerArtworksMutationSuccessDetails
```

Comment:
Naming here as well, the field name might be better off as something besides `partnerArtworks`.

Reply:
Updated to `partnerArtworksBulkEdit`